docs/standard/native-interop/interop-exceptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Platform Neutral Interop Exceptions"
+ms.date: "01/16/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "unmanaged code, exceptions"
+  - "exceptions, unmanaged code"
+  - "interop, exceptions"
+  - "exceptions, interop"
+  - "stack, jump"
+  - "jump, stack"
+description: Information the use of unmanaged code interop exceptions.
+---
+# Working with Interop Exceptions in Unmanaged Code
+Unmanaged code exception interop is not supported due to the portability issues that arise, since managed code can't know how these exception mechanisms work under the covers in non-Windows platforms.
+
+## Longjump Behaviors
+Longjump interop is held to the same constraints as exception handling and is only supported on Windows.
```

Comment:
As I have mentioned in to other comment, I would document setjmp/longjump as unsupported on all platforms. I do not think we can declare as supported with confidence.